Automattic/wp-calypso
# Post-purchase: Enable user to cancel a migration manually

Related to #96157
Depends on #95963

## Proposed Changes
* Add a button to cancel a pending migration
* Reloads the site after the cancellation. 


> [!IMPORTANT]  
> As demonstrated in the video above, the page header is not listening to the act of reloading the site details, so it shows the pending label until the user reloads the page.  It will be fixed on the issue  #96230 


https://github.com/user-attachments/assets/452564de-59cb-433d-baf5-9fa36092f384



## Testing Instructions
* Enable the feature flag, sessionStorage.setItem('flags', 'automated-migration/pending-status');
* Set a site with one of the pending status (`migration-pending-diy` or `migration-pending-difm`)  using any of the migration flows and stopping on the credential step(DIFM) or the instructions step(DIY)
* Go to `/site` .
* Click on the site with the `pending` label.
* Click on the cancel button.
* Reload the page and check there is no pending label. 


## Pre-merge Checklist
